cdc/async_sink.go

```diff
@@ -0,0 +1,185 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cdc
```

Comment:
Please add a TODO to remove this file once new owner is enabled.

Reply:
Ok, but seems that all old owner related code should be cleaned while old owner is enabled.

Reply:
done